model/project.go

```diff
@@ -94,18 +93,23 @@ type BuildVariantTaskUnit struct {
 	// the project level, or an error will be thrown
 	Name string `yaml:"name,omitempty" bson:"name"`
 	// IsGroup indicates that it is a task group. This is always populated for
-	// task groups after translating the parser project to the project.
+	// task groups after project translation.
 	IsGroup bool `yaml:"-" bson:"-"`
 	// IsPartOfGroup indicates that this unit is a task within a task group. If
-	// this is set, then GroupName is also set. Note that project translation
-	// does not expand task groups into their individual tasks, so this is only
-	// set for special functions that explicitly expand task groups into
-	// individual task units (such as FindAllBuildVariantTasks).
+	// this is set, then GroupName is also set.
```

Comment:
If this is only true for IsPartOfGroup and GroupName, it might be reasonable to move these to the bottom and just say the comment once for both fields, but whichever works.

Reply:
Both are part of the same special functions, but I didn't consolidate because Go code inspection tools (like the ones to automatically show docs for a field in IDEs) are only smart enough to display the one comment directly above the field. Duplicating the note makes sure that readers can't miss the special-ness comment by only reading the doc for one field + makes it obvious that the special case is true for specifically just these two fields.